lago/cmd.py

```diff
@@ -767,6 +756,20 @@ def check_group_membership():
         warnings.warn('current session does not belong to lago group.')
 
 
+def check_deps():
+    try:
+        # Checks that all the deps are installed
+        pkg_resources.require("lago")[0]
+    except pkg_resources.ContextualVersionConflict as e:
+        # Hack that allows to run stevedore without checking
+        # for it's dep. it is required for systems running stevedore 1.1.0
+        # and pbr > 1.
+        LOGGER.debug(traceback.format_exc())
```

Comment:
It shouldn't be in debug level, as his exception should always be logged.
In general - I think we rather use logging.exception('message') for exceptions. see: https://docs.python.org/2/library/logging.html#logging.Logger.exception.